mkdocs.yml

```diff
@@ -51,6 +52,7 @@ nav:
       - Sidecars: docs/developing/sidecars.en.md
       - Storage: docs/developing/storage.en.md
       - Task Definition Overrides: docs/developing/taskdef-overrides.en.md
+      - Publish/Subscribe: docs/developing/publish-subscribe.en.md
```

Comment:
What do you think of moving this below "Service Discovery" as they're related to service-to-service communication?